internal/lvmd/util.go

```diff
@@ -0,0 +1,11 @@
+package lvmd
+
+import "math"
+
+func calcThinPoolFreeBytes(overProvisionRatio float64, tpuSizeBytes, tpuVirtualBytes uint64) uint64 {
+	poolSize := uint64(math.Floor(overProvisionRatio * float64(tpuSizeBytes)))
```

Comment:
I wonder if `poolSize` really represents the pool size.
IIUC, `tpuSizeBytes` is the pool size, so we need another name for`uint64(math.Floor(overProvisionRatio * float64(tpuSizeBytes)))`.

How do you think about changing it to something like `virtualPoolSize`, `effectivePoolSize`, or `overprovisionedPoolSize`?

Reply:
Thank you for your review. I've just fixed this, so could you review this PR again?